app/views/development_metrics/departments.erb

```diff
@@ -26,32 +26,36 @@
           <%= render 'development_metrics/code_climate/code_climate_department_summary',
           department: @department, code_climate: @code_climate %>
         </div>
-        <% if @review_turnaround && @merge_time %>
-          <div class="row">
-            <div class="metrics">
-              <div class="graph">
-                <%= render 'development_metrics/review_turnaround/main_metric', review_turnaround: @review_turnaround %>
-              </div>
-              <div class="graph details">
-                <%= render 'development_metrics/review_turnaround/details_metric', review_turnaround: @review_turnaround %>
-              </div>
+        <% if @review_turnaround && @merge_time && @pull_request_size %>
```

Comment:
when are any of these 3 going to not be present? is it possible that only one of them is not preset but the other two are? in that case should we show something instead?

Reply:
It's a good question. I'll check it and change it if it does not make sense.

Reply:
As how things work right now, these three can be either all empty or all not empty, simultaneously. And they are empty when no department has been selected.
I think we can safely change it to check only if the department has been selected.
Good catch! 👏